content/content-tool/tool/src/java/org/sakaiproject/content/tool/ResourcesAction.java

```diff
@@ -4215,7 +4215,7 @@ public String buildListContext (	VelocityPortlet portlet,
 			// notshow the public option or notification when in dropbox mode
 			context.put("dropboxMode", Boolean.TRUE);
 			// allow filtering of dropboxes by group (SAK-14625)
-			String collectionId = (String) state.getAttribute (STATE_COLLECTION_ID);
```

Comment:
Hi Daniel, can you update the PR?

One more note on leaving comments :) you should leave a comment if its useful. Useless comments is just clutter. For the most part you should aim to write code that is self explanatory and then you don't need comments. Also with most IDE's you can just select "Show Annotations" and then every line shows the commit message and JIRA issue.